ui/lib/core/addon/components/form-field.js

```diff
@@ -42,6 +42,23 @@ import { dasherize } from 'vault/helpers/dasherize';
  */
 
 export default class FormFieldComponent extends Component {
+  // alphabetical list of all editTypes if they hide <FormFieldLabel>
+  shouldHideLabel = {
+    boolean: true,
+    file: true,
+    json: true,
+    kv: true,
+    mountAccessor: true,
+    optionalText: true,
+    password: false,
+    radio: false,
+    regex: true,
+    searchSelect: true,
+    stringArray: true,
+    textarea: false,
+    ttl: true,
+    yield: false, // TODO for flexibility change to false?
```

Comment:
this is a new editType added by the PKI work so added this "to do" if it felt reasonable while implementing the redesign